osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerMatchSubScreen.cs

```diff
@@ -53,7 +53,7 @@ public MultiplayerMatchSubScreen(Room room)
         [BackgroundDependencyLoader]
         private void load()
         {
-            InternalChildren = new Drawable[]
+            AddRangeInternal(new Drawable[]
```

Comment:
uneasy with changes like this. what's the explanation for changing this in this way? if it's to avoid removing a component that's added in a base class, why does this not implement the `content` pattern properly to avoid accidentally removing that from hierarchy?

Reply:
well, this is a screen, no `Container` inherited, overriding `AddInternal`, `RemoveInternal`, `ClearInternal` to throw exception warning about not using a certain `Content` property is a way but I think it's kinda over the top, what do you think?

Reply:
then make a protected container? virtual method? anything?

the fact that this is a screen actually makes doing this as you have it worse, not better, from a code quality/usability standpoint.

Reply:
went ahead and have overriden all internal methods to mutate a certain container instead

Reply:
I've changed this to just disallow ClearInternal. Should be safe enough while not leading to shoddy code.